app/Models/ActivityUiUpdates.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ActivityUiUpdates extends Model
```

Comment:
If we want to have 1 field then it's better to have that field for activity_type table itself. no need for the extra table.